### ISSUE TITLE
subversion build error on 10.12 preview

### ISSUE THREAD

**Issue by leaxoy** [Homebrew/homebrew-core#3489]
https://gist.github.com/leaxoy/0e3ef487396c1ad41c445ad31e7586c6

**Comment by DomT4** [comment]
Apparently we've lost `apr-1-config` and `apu-1-config` in 10.12, but the libraries & headers remain. The problem with that being almost everything has come to rely on the `-config` executables to find the libraries/headers, so now things are breaking. Apple enjoys half removing bits for no obvious reason 😕.

This will require some core & formula rewriting. Give me a little bit to throw things together.

**Comment by DomT4** [comment]
I went for a more limited tweak to purely fix `subversion` for now. I want to get a bit further along in the macOS Sierra development timetable before leaping to conclusions on what Apple is going to do, and how upstreams will react here.

It's kind of a pain that most things can hunt the APR & APU elements themselves without leaning on the `*-config` executables that were removed in 10.12, but the `serf` element within the `subversion` formula bails without that present.

Should be fixed after a `brew update` from testing, but shout if you hit any issues with it. Thanks for the report!
